images/bash/config/main.tf

```diff
@@ -0,0 +1,20 @@
+variable "extra_packages" {
+  description = "Additional packages to install."
+  type        = list(string)
+  default     = []
+}
+
+output "config" {
+  value = jsonencode({
+    contents = {
+      packages = concat([
+        "bash",
+        "busybox",
+        "curl",
+      ], var.extra_packages)
```

Comment:
did you conflate these two changes?

Reply:
I based this change on the other one (#1444) on accident, but it was nice to see it trigger bash's leg.